certbot/docs/install.rst

```diff
@@ -61,6 +61,21 @@ Alternate installation methods
 If you are offline or your operating system doesn't provide a package, you can use
 an alternate method for installing ``certbot``.
 
+.. _snap-install:
+
+Snap
+----
+
+Most modern Linux distributions (basically any that use systemd) can install
+Certbot packaged as a snap. Support for the Certbot snap is currently in its
+beta phase and limited the x86_64 architecture, but it provides an easy way to
+ensure you have the latest version of Certbot with features like automated
+certificate renewal preconfigured.
+
+You can find instructions for installing the Certbot snap at
+https://certbot.eff.org/instructions by selecting your server software and then
+choosing "snapd" in the "System" dropdown menu.
```

Comment:
maybe add the same parenthetical I suggested here: https://github.com/certbot/certbot/pull/7890#discussion_r412508926